astropy/table/serialize.py

```diff
@@ -17,7 +17,8 @@
                              'astropy.coordinates.distances.Distance',
                              'astropy.coordinates.earth.EarthLocation',
                              'astropy.coordinates.sky_coordinate.SkyCoord',
-                             'astropy.table.table.NdarrayMixin')
+                             'astropy.table.table.NdarrayMixin',
+                             'sunpy.coordinates.representation.Longitude180')
```

Comment:
This seems confusing to me given that SunPy is not listed as optional dependency of Astropy. I feel like this should be within SunPy, not in core.

Reply:
It does seem like astropy should have a public registry for these that Sunpy could add to

Reply:
I don't disagree, any chance we could get this into a 2.0.x bug fix and I will add a registry for 3.0? It stops us round tripping coords to disk, which is really annoying.

Reply:
Putting into 2.0.x will require a stretch of what could be called a bug fix.  I guess I would label this a highly desired backport and then have a policy discussion on if / when they are allowed.

Reply:
As is, I agree, but allowing subclasses serves the purposes and could be more easily argued to be a bug fix. It does mean, independently, that sunpy would only work with astropy >=2.0.3. But since it is a LTS, that may still be very useful.